_includes/authorizations-resource.md

```diff
@@ -0,0 +1,184 @@
+{% assign payment-instrument = include.payment-instrument | default: creditcard %}
+
+### Authorizations
+
+The `authorizations` resource will list the authorization transactions
+made on a specific payment.
+
+{:.code-header}
+**Request**
+
+```http
+GET /psp/{{payment-instrument}}/payments/<payment-id>/authorizations HTTP/1.1
+Host: api.externalintegration.payex.com
+Authorization: Bearer <AccessToken>
+Content-Type: application/json
+```
+
+{:.code-header}
+**Response**
+
+```http
+HTTP/1.1 200 OK
+Content-Type: application/json
+
+{
+  "payment": "/psp/{{payment-instrument}}/payments/<payment-id>",
+  "authorizations": {
+    "id": "/psp/{{payment-instrument}}/payments/<payment-id>/authorizations",
+    "authorizationList": [
+      {
+        "id": "/psp/{{payment-instrument}}/payments/<payment-id>/authorizations/<transaction-id>",
+        "consumer": {
+          "id": "/psp/{{payment-instrument}}/payments/<payment-id>/consumer"
+        },
+        "legalAddress": {
+          "id": "/psp/{{payment-instrument}}/payments/<payment-id>/legaladdress"
+        },
+        "billingAddress": {
+          "id": "/psp/{{payment-instrument}}/payments/<payment-id>/billingaddress"
+        },
+        "transaction": {
+          "id": "/psp/{{payment-instrument}}/payments/<payment-id>/transactions/<transaction-id>",
+          "created": "2016-09-14T01:01:01.01Z",
+          "updated": "2016-09-14T01:01:01.03Z",
+          "type": "Authorization",
+          "state": "Initialized|Completed|Failed",
+          "number": 1234567890,
+          "amount": 1000,
+          "vatAmount": 250,
+          "description": "Test transaction",
+          "payeeReference": "AH123456",
+          "failedReason": "",
+          "isOperational": false,
+          "operations": [
+            {
+              "href": "https://api.externalintegration.payex.com/psp/{{payment-instrument}}/payments/<payment-id>",
+              "rel": "edit-authorization",
+              "method": "PATCH"
+            }
+          ]
+        }
+      }
+    ]
+  }
+}
+```
+
+#### Create Authorization transaction
+
+To create an `authorization` transaction, perform the `create-authorization`
```

Comment:
Wouldn't it be nice if the previous code example contained the `create-authorization` operation?

Reply:
Such a feature has been added